.github/workflows/ci.yml

```diff
@@ -83,6 +85,17 @@ jobs:
       custom-requirements: ''
     secrets: inherit
 
+  upload-development-docs:
+    runs-on: ubuntu-latest
+    if: startsWith(github.head_ref, 'master') || github.event.action == 'ready_for_review' || !github.event.pull_request.draft
```

Comment:
@jorgepiloto don't we want the dev doc to only be updated when pushing to master?

Reply:
or is that for testing now?

Reply:
You are right, @PProfizi. Thanks for pointing this out. Let me update the conditional for triggering this job.